icons.toml

```diff
@@ -0,0 +1,19 @@
+name = "default"
+
+# All icons here must be available as [default Unicode characters](https://en.wikipedia.org/wiki/List_of_Unicode_characters)
+
+[diagnostic]
+error = {icon = "●"}
+warning = {icon = "●"}
+info = {icon = "●"}
+hint = {icon = "●"}
+
+[breakpoint]
+verified = {icon = "●"}
+unverified = {icon = "◯"} 
+pause-indicator = {icon = "▶"}
+
+[diff]
+added = {icon = "▍"}
+deleted = {icon = "▔"}
+modified = {icon = "▍"}
```

Comment:
To work around this config (where we want ASCII by default, icons if supported), I'd add a toplevel config setting `icons = true` that would switch over from ascii symbols to nerdfonts. There's precedence in `true-color` support working this way for example.